cmd/wallet-web/src/pages/CHAPISharePage.vue

```diff
@@ -4,6 +4,113 @@
  * SPDX-License-Identifier: Apache-2.0
 -->
 
+<script setup>
+import { ref, inject, computed } from 'vue';
+import {
+  filterCredentialsByType,
+  getCredentialType,
+  getCredentialDisplayData,
+  getCredentialIcon,
+  WalletGetByQuery,
+} from '@/mixins';
+import SpinnerIcon from '@/components/icons/SpinnerIcon.vue';
+import { useI18n } from 'vue-i18n';
+
+// Local Variables
+const errors = ref([]);
+const requestOrigin = ref('');
+const loading = ref(true);
+const sharing = ref(false);
+const credsFound = ref([]);
+const issuersFound = ref([]);
+const credentialDisplayData = ref('');
+const manifestCredType = 'IssuerManifestCredential';
+const wallet = ref(null);
+const presentation = ref(null);
+
+// Computed
+const showCredentialsMissing = computed(() => {
+  return credsFound.value.length === 0;
+});
```

Comment:
When your lambda function only returns an expression, you can simplify it as the following:
```suggestion
const showCredentialsMissing = computed(() => credsFound.value.length === 0);
```

Reply:
Got it!

Reply:
And my bad about the errors... I thought I compiled it.... Will fix!